### DISCUSSION TITLE
Use both Internal and External Bus?

### DISCUSSION THREAD

**Discussion by ejsmith** [BrighterCommand/Brighter#2475] [Q&A]
Is it possible to use both the internal and external bus and configure specific messages to use external? I didn't see anything in the docs for this. I basically want most of my application to use internal mediator pattern, but a select few would be handler externally (service bus).

**Comment by DevJonny** [comment]
Hi Eric. 

Yes, this is absolutely possible if I understand your question correctly. Commands and Events dispatched via the `.Send[Async](..)` or `Publish[Async](..)` methods respectively will use the internal Command Processor. Commands and Events dispatched via the `.Post[Async](..)` or `.DepositPost[Async](..)` methods will be sent out via the external bus.
